styles/globals.css

```diff
@@ -264,6 +264,14 @@
     @apply self-center;
   }
 
+  .about-container .md p {
+    @apply mb-4 text-2xl leading-relaxed;
+  }
+
+  .about-container .md a {
+    text-decoration: none;
+  }
+
```

Comment:
Can't we apply these to the markdown file in about rather than making a class? Since it wont be used anywhere other than about page.

Reply:
Hmmm. I would say its on the author to follow styling consistency. We should avoid adding specific classes in globals.css imo. Larger globals.css file also affects perf.